actors/user/src/test/java/org/sunbird/user/AddressManagementActorTest.java

```diff
@@ -0,0 +1,137 @@
+package org.sunbird.user;
+
+import static akka.testkit.JavaTestKit.duration;
+import static org.powermock.api.mockito.PowerMockito.mock;
+import static org.powermock.api.mockito.PowerMockito.when;
+
+import akka.actor.ActorRef;
+import akka.actor.ActorSystem;
+import akka.actor.Props;
+import akka.testkit.javadsl.TestKit;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mockito;
+import org.powermock.api.mockito.PowerMockito;
+import org.powermock.core.classloader.annotations.PowerMockIgnore;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+import org.powermock.modules.junit4.PowerMockRunner;
+import org.sunbird.cassandra.CassandraOperation;
+import org.sunbird.cassandraimpl.CassandraOperationImpl;
+import org.sunbird.common.ElasticSearchUtil;
+import org.sunbird.common.models.response.Response;
+import org.sunbird.common.models.util.JsonKey;
+import org.sunbird.common.models.util.datasecurity.EncryptionService;
+import org.sunbird.common.request.Request;
+import org.sunbird.helper.ServiceFactory;
+import org.sunbird.user.actors.AddressManagementActor;
+
+@RunWith(PowerMockRunner.class)
+@PrepareForTest({
+  ServiceFactory.class,
+  ElasticSearchUtil.class,
+  EncryptionService.class,
+  org.sunbird.common.models.util.datasecurity.impl.ServiceFactory.class
+})
+@PowerMockIgnore({"javax.management.*"})
+public class AddressManagementActorTest {
+
+  private static ActorSystem system = ActorSystem.create("system");
+  private static final Props props = Props.create(AddressManagementActor.class);
+  private CassandraOperation cassandraOperation;
+
+  @Before
+  public void beforeEachTest() throws Exception {
+
+    PowerMockito.mockStatic(ServiceFactory.class);
+    PowerMockito.mockStatic(org.sunbird.common.models.util.datasecurity.impl.ServiceFactory.class);
+    EncryptionService encryptionService = Mockito.mock(EncryptionService.class);
+    Mockito.when(
+            org.sunbird.common.models.util.datasecurity.impl.ServiceFactory
+                .getEncryptionServiceInstance(null))
+        .thenReturn(encryptionService);
+
+    try {
+      Mockito.when(encryptionService.encryptData(Mockito.anyString())).thenReturn("encrptUserId");
+    } catch (Exception e) { // TODO Auto-generated catch block
+      Assert.fail("Initialization failed");
+    }
+    cassandraOperation = mock(CassandraOperationImpl.class);
+    when(ServiceFactory.getInstance()).thenReturn(cassandraOperation);
+  }
+
+  @Test
+  public void testInsertAddressSuccess() {
+    TestKit probe = new TestKit(system);
```

Comment:
again there is a pattern in each -testcase - one common function receiving the 
(String requestObjType, boolean success, Response response)

Reply:
yes so?

Reply:
so create such function -> and call it from each test-case -> test-code will be reduced quite a bit